### PR TITLE
feat: add font color setting supports

### LINKED ISSUE #19: 建议支持更改字体颜色和背景色
建议支持更改字体颜色和背景色

### PR DESCRIPTION
**Pull request by ruibaby**
支持为文字设置颜色。

/kind feature

Fixes #19 

image
image

```release-note
支持修改文字颜色和背景色
```

### CODE REVIEW

File: packages/editor/src/extensions/color/index.ts [hunk 1]
```diff
@@ -20,7 +21,7 @@ const Color = TiptapColor.extend<ColorOptions & ExtensionOptions>({
             editor,
             isActive: false,
             icon: markRaw(MdiFormatColor),
-            title: "字体颜色",
+            title: i18n.global.t("core.common.color"),
```
Comment (JohnNiang) [inline packages/editor/src/extensions/color/index.ts]: `core` or `editor`?

Comment (ruibaby) [reply packages/editor/src/extensions/color/index.ts]: Fixed

**Review by guqing — COMMENTED** [review]
/lgtm

**Review by JohnNiang — APPROVED** [review]
It works very well.
image

/approve

**Pull request merged**